pyproject.toml

```
@@ -39,3 +39,18 @@ known-third-party = [
   "opencensus",
 ]
 
+# https://github.com/microsoft/pyright/blob/main/docs/configuration.md#type-check-rule-overrides
+[tool.pyright]
+typeCheckingMode = "strict"
+reportUnnecessaryTypeIgnoreComment = true
+reportMissingTypeStubs = false
```

```suggestion
reportMissingTypeStubs = false
pythonVersion = "3.8"
```
to run with the lowest supported version?

---

This is not needed.

---

If not set it will run with the default in CI, like 3.11 no?

---

Ah, I see. Where's that version coming from?

---

From jinja2 templates which generates the workflows.